Doesn't look related:
```
13:13:43 /Users/osrf/jenkins-agent/workspace/ci_osx/ws/src/ros2/rviz/rviz_common/include/rviz_common/ros_integration/ros_node_abstraction_iface.hpp:45:7: error: definition of type 'Executor' conflicts with type alias of the same name
13:13:43 class Executor;
13:13:43       ^
13:13:43 /Users/osrf/jenkins-agent/workspace/ci_osx/ws/install/rclcpp/include/rclcpp/executor.hpp:377:7: note: 'Executor' declared here
13:13:43 using Executor [[deprecated("use rclcpp::Executor instead")]] = rclcpp::Executor;
13:13:43       ^
13:13:43 1 error generated.
```

--- a/rviz_common/CMakeLists.txt
+++ b/rviz_common/CMakeLists.txt
@@ -24,12 +24,6 @@ endif()
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
-  # TODO(wjwwood): try to remove this -- currently needed to pass on CI
-  include(CheckCXXCompilerFlag)
-  check_cxx_compiler_flag(-Wno-gnu-zero-variadic-macro-arguments HAS_W_FLAG)
-  if(HAS_W_FLAG)
-    add_compile_options(-Wno-gnu-zero-variadic-macro-arguments)
-  endif()
 endif()
 
 find_package(ament_cmake REQUIRED)

